src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -140,6 +140,20 @@ protected override Task CreateVersionsAsync(CancellationToken cancellationToken)
             var latestPrerelease = allVersionsAllowed.FirstOrDefault(v => v.version.IsPrerelease);
             var latestStableVersion = allVersionsAllowed.FirstOrDefault(v => !v.version.IsPrerelease);
 
+            foreach (IProjectContextInfo project in _nugetProjects)
+            {
+                if (project.ProjectStyle.Equals(ProjectModel.ProjectStyle.PackageReference) && installedDependency != null)
+                {
+                    VersionRange installedVersionRange = VersionRange.Parse(installedDependency?.VersionRange?.OriginalString, true);
+                    NuGetVersion bestVersion = installedVersionRange.FindBestMatch(allVersionsAllowed.Select(v => v.version));
+                    _versions.Add(new DisplayVersion(installedVersionRange, bestVersion, additionalInfo: null));
+                }
+                else if (installedDependency != null)
+                {
+                    _versions.Add(new DisplayVersion(VersionRange.Parse(installedDependency?.VersionRange?.OriginalString, false), additionalInfo: null));
```

Comment:
I get that this PR isn't the complete feature, end to end, there will be others. But `_versions` is a variable defined in this method, and never used anywhere else. What's the point addings values to this list?

Also, identifiers starting with `_` should only be used for class private fields, nothing else. So not a method-local variable, although I know the variable wasn't added in this PR.